integration-test/src/test/java/com/datastrato/gravitino/integration/test/util/spark/SparkTableInfo.java

```diff
@@ -62,9 +94,33 @@ static SparkTableInfo create(SparkBaseTable baseTable) {
             .collect(Collectors.toList());
     sparkTableInfo.comment = baseTable.properties().remove(ConnectorConstants.COMMENT);
     sparkTableInfo.tableProperties = baseTable.properties();
+    Arrays.stream(baseTable.partitioning())
+        .forEach(
+            transform -> {
+              if (transform instanceof BucketTransform
+                  || transform instanceof SortedBucketTransform) {
+                sparkTableInfo.setBucket(transform);
+              } else if (transform instanceof IdentityTransform) {
+                sparkTableInfo.addPartition(transform);
+              } else {
+                throw new NotSupportedException("Not support Spark transform: " + transform.name());
+              }
+            });
     return sparkTableInfo;
   }
 
+  public List<SparkColumnInfo> getNonPartitionColumns() {
```

Comment:
According the name style, this should be called `getUnPartitionedColumns`.

Reply:
done